tests/suites/test_suite_x509write.function

```diff
@@ -93,6 +93,10 @@ void x509_csr_check( char * key_file, char * cert_req_check_file, int md_type,
 
     memset( &rnd_info, 0x2a, sizeof( mbedtls_test_rnd_pseudo_info ) );
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Please use `USE_PSA_INIT( )`, and `USE_PSA_DONE( )` in the cleanup. They'll do the right thing to initialize PSA crypto if needed and check that resources are properly deallocated.